scripts/exclude-links.txt

```diff
@@ -1,41 +1,16 @@
-http://127.0.0.1:10000/devstoreaccount1;
-http://localhost:8000/
-http://millionsongdataset.com/pages/getting-dataset/
-https://$
-http://s3-external-1.amazonaws.com/bucket/path
-https://accounts.google.com/o/oauth2/auth
-https://api.cloudflare.com/client/v4/zones/$
-https://circleci.com/gh/iterative/dvc.org
-https://discuss.$
-https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA
```

Comment:
did we change rules a bit?

Reply:
Yeah, the scraper on the new checker specifically inspects `[]()`-style links on markdown files, and doesn't go through code blocks like the bash one did. I checked, and most of these exclusions are from code blocks- see the ones ending in `$`: those are string interpolations.

Reply:
sounds good!